app/features-json/api_controller.rb

```diff
@@ -107,13 +107,29 @@ def jwt
 
         return payload
       rescue JWT::InvalidIssuerError
-        halt(403, { "Content-Type" => "text/plain" }, "The token does not have a valid issuer.")
+        json_error!(
+          error_message: "The token does not have a valid issuer",
+          error_key: "Authentication.Token.InvalidIssuer",
+          error_code: 403
+        )
       rescue JWT::InvalidIatError
-        halt(403, { "Content-Type" => "text/plain" }, 'The token does not have a valid "issued at" time.')
+        json_error!(
+          error_message: "The token does not have a valid 'issued at' time",
+          error_key: "Authentication.Token.MissingIssuedTime",
```

Comment:
Since we use the string `Authentication.Token` a lot, it would be great to have some way of building these error keys that is less likely to be prone to typos.